app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :users
```

Comment:
Sorry for the late reply.

Read this article it will help you

https://www.microverse.org/blog/how-to-use-associations-and-class-names-in-ruby-on-rails